examples/drake_walking.ipynb

```diff
@@ -250,7 +250,7 @@
     "            break\n",
     "\n",
     "    # Reading new references\n",
-    "    com, dcom, forces_left, forces_right = mpc.get_references()\n",
+    "    com, dcom, forces_left, forces_right, ang_mom = mpc.get_references()\n",
```

Comment:
This seems not related to the change of the PR, is it a leftover ?

Reply:
Yes, when I pulled the latest version of comodo, I got a `too many to unpack` error. I saw the `mujoco` to see that this was added. So corrected it. You are right, it is not relevant to the migration to `amo_urdf`